java-pinning-aar/src/main/java/eu/geekplace/javapinning/pin/Pin.java

```diff
@@ -123,6 +106,25 @@ public static Pin fromString(String string) {
 	}
 
 	/**
+	 * Create a new "plain certificate" {@link Pin} from the given {@link X509Certificate}.
+	 * @param certificate the certificate to create a {@link Pin} for.
+	 * @return the {@link Pin} for the given certificate.
+	 * @throws CertificateEncodingException if the given certificate could not be encoded.
+	 */
+	public static CertPlainPin fromCertificate(X509Certificate certificate) throws CertificateEncodingException {
```

Comment:
I think that this should be a method of CertPlainPin. That makes usages of that static method more readable. Compare

```java
foo = Pin.FromCertificate(bar)
```

vs

```java
foo = CertPlainPin.fromCertificate(bar)
```